ingestion/src/metadata/ingestion/api/processor.py

```diff
@@ -41,7 +41,11 @@ class Processor(Closeable, Generic[Entity], metaclass=ABCMeta):
     @classmethod
     @abstractmethod
     def create(
-        cls, config_dict: dict, metadata_config_dict: dict, ctx: WorkflowContext
+        cls,
+        config_dict: dict,
+        metadata_config_dict: dict,
+        ctx: WorkflowContext,
+        **kwargs
```

Comment:
Allows us to pass more parameters to the processor, such as the `session` in the ORM Processor